tests/bench/test_bench_cli.py

```
@@ -0,0 +1,68 @@
+    assert expected_set.issubset(args_set), (
+        f"{expected_set=} " f"not in {args_set=}"
+    )
+
+
+def test_help(capfd: pytest.CaptureFixture) -> None:
+    help_line: str = "--help"
+    with pytest.raises(SystemExit) as caught:
+        success: bool = _main(*help_line.split())
+        assert not success
+        assert caught.value.code == 0
+        assert caught.type == SystemExit
```

Ruff

`pytest.raises()` block should contain a single simple statement

(PT012)